src/REQUIRE

```diff
@@ -0,0 +1,4 @@
+julia 0.6
+DifferentialEquations
+LsqFit
+Optim
```

Comment:
Can this file be deleted now?

Reply:
For sure. I got rid of the main one (fb0612c979dcf157d25ce208b7cdfdeb18ae253c) so I'm not sure how we ended up with another `REQUIRE` in `src`.